src/gasman.c

```diff
@@ -554,7 +552,7 @@ static inline BOOL IS_BAG_ID(void * ptr)
 
 BOOL IS_VALID_BAG_ID(Bag bag)
 {
-    return IS_BAG_ID(bag) && (PTR_BAG(bag) >= MptrEndBags);
+    return IS_BAG_ID(bag) && (PTR_BAG(bag) >= OldBags);
```

Comment:
@ChrisJefferson FYI this should not cause problems in regular usage, but it should catch a few more potential memory corruptions (note in particular that for unused master pointers, `PTR_BAG` points to the next unused master pointer, in which case there is a chance the old `IS_VALID_BAG_ID` might have succeeded; while the modified version should catch it.

Reply:
I had a check and it didn't seem to find anything (currently), but also didn't break anything